### PR TITLE
New ALB-ECS templates

### PR DESCRIPTION
**Pull request by cb17**

### CODE REVIEW

File: examples/ALB-ECS-WithElasticSearchLogs-AppStack [hunk 1]
```diff
@@ -0,0 +1,344 @@
+from troposphere import Join, GetAtt, ImportValue
```
Comment (phobologic) [inline examples/ALB-ECS-WithElasticSearchLogs-AppStack]: This should be a `.py` file, like the rest in the `examples` directory

File: examples/ALB-ECS-WithElasticSearchLogs-NetworkStack [hunk 2]
```diff
@@ -0,0 +1,540 @@
+import troposphere.elasticloadbalancingv2 as elb
```
Comment (phobologic) [inline examples/ALB-ECS-WithElasticSearchLogs-NetworkStack]: This file name should end with `.py` (`ALB-ECS-WithElasticSearchLogs-NetworkStack` -> `ALB-ECS-WithElasticSearchLogs-NetworkStack.py`)

**Comment by kienpham2000** [comment]
this is awesome guys, I just used this example to launch an ECS cluster with ALB!

**Comment by phobologic** [comment]
Ping @cb17 - any chance you can update this?

**Comment by patrickpierson** [comment]
I was able to get the APP stack to work. The network stack is having issues on the later versions of troposphere. Given that @cb17 hasnt responded. I think this is closable and we could use the APP stack example to close this in another PR. Same for https://github.com/cloudtools/troposphere/pull/640

**Comment by cb17** [comment]
@patrickpierson @phobologic Hi guys, sorry for the radio silence. I had been meaning to circle back around to this, but got a little disconnected from these templates. FWIW, the code for the ElasticSearch JS Lambda function I just stole from what gets generated in the console (and minified it, and added a "var esdomain = Ref(ESDomain)" to the front of it). So if someone wants to add that back in, I'm sure it could be useful to someone.

**Comment by patrickpierson** [comment]
I think that since this PR is over a year old it makes sense to close it and if you want to create a new one based on the newer version of troposphere it would be good.

**Pull request closed without merging**